addons/mail/static/src/core/common/message.js

```diff
@@ -424,8 +424,18 @@ export class Message extends Component {
     prepareMessageBody(element) {}
 
     enterEditMode() {
+        const body = document.createElement("span");
+        body.innerHTML = this.props.message.body;
+        const mentionedChannelElements = body.querySelectorAll(".o_channel_redirect");
+        const mentionedChannels = Array.from(mentionedChannelElements).map((el) =>
+            this.store.Thread.insert({
+                id: el.dataset.oeId,
+                model: el.dataset.oeModel,
+            })
```

Comment:
id/model are user input, I don't like that we insert them without checking their existence.
I see why get is not enough in this case (channel not yet known but valid), but it should be `getOrFetch` in master (and whatever in version already has this function).

Reply:
will do a fix again then